Prüfschritte/de/11.1.2.3 Audiodeskription oder Volltext-Alternative für Videos.adoc

```diff
@@ -4,96 +4,64 @@ include::include/attributes.adoc[]
-Video keine Tonspur hat.
+==== Art der Einbindung ermitteln
 
-Verzichtbar ist die Audiodeskription auch für Videos, die lediglich als
-Medienalternative zu einem textbasierten Inhalt dienen, das heißt ergänzend
-zu einem Text angeboten werden, um den Textinhalt zusätzlich in anderer Form
-zu vermitteln.
-Dies gilt nur für Videos, die keine über den Textinhalt hinausgehende
-Informationen bieten und die klar als Medienalternative zum Text erkennbar sind.
-
-Gebärdensprachvideos brauchen keine Audiodeskription.
+Die Audiodeskription kann auf verschiedenem Wege angeboten werden:
 
-=== Prüfung
+* Die Audiodeskription ist bereits in die normalen Tonspur integriert.
+* Eine weitere Version des Videos mit Audiodeskription wird angeboten. Wenn das der Fall ist, wird diese Version wird geprüft. Es muss dann ein funktionierender Link zu dieser Version im unmittelbaren Kontext des Videos angeboten werden, ebenso wie ein Zurück-Link.
+* Das Video hat eine zuschaltbare Tonspur mit der Audiodeskription. Diese Spur wird zusätzlich eingeschaltet.
+* Das Video wird in der App oder in einem externen, vom Format abhängigen Video-Player abgespielt.
 
 ==== Audiodeskription prüfen
 
-Es wird eine Sicht- und Hörprüfung vorgenommen.
+. Video abspielen und Bildinhalte daraufhin überprüfen, ob wichtige Informationen vorhanden sind, die sich nicht auch aus dem Ton entnehmen lassen.
+. Zu häufigen informationstragenden Elementen der Bildebene gehören zum Beispiel Einblendungen von sprechenden Personen. Ist es dann wichtig, mitzubekommen, wer spricht? Sind visuelle Informationen auch in der Tonspur vorhanden? 
+. Ist das Video lediglich eine Medienalternative zu einem textbasierten Inhalt, wird es also ergänzend zu einem Text angeboten? In diesem Fall ist keine Audiodeskription erforderlich. Dies gilt allerdings nur dann, wenn das Video keine Inhalte enthält, die inhaltlich über die Textversion hinausgehen. 
 
-Art der Einbindung feststellen.
-Die Audiodeskription kann auf verschiedenem Wege angeboten werden:
 
-* Die Audiodeskription ist bereits in der normalen Tonspur enthalten.
-* Eine weitere Version des Videos mit Audiodeskription wird angeboten.
-  Diese Version wird geprüft.
-  Ein funktionierender Link zu dieser Version muss im unmittelbaren Kontext des
-  Videos angeboten werden, ebenso wie ein Zurück-Link.
-* Das Video hat eine zuschaltbare Tonspur mit der Audiodeskription.
-  Diese Spur wird zusätzlich geschaltet.
-* Das Video wird In der App oder in einem externen, vom Format abhängigen
-  Video-Player abgespielt.
+==== 2.2 Prüfung von Volltextalternativen
 
-Wenn eine Audiodeskription vorhanden ist, wird eine kurze Sequenz der
-Deskription angehört, um stichprobenartig festzustellen, ob sie nutzbar ist.
+Es wird eine Sicht- und Hörprüfung vorgenommen. Das Video wird in der App oder in einem externen, vom Format abhängigen Video-Player abgespielt. Die Volltextalternative sollte folgende Informationen enthalten:
```

Comment:
In der Regel ja in der App - Hinweis auf externen Player hier erstmal weglassen? Oder gibt es solche Fälle?

Reply:
Hinweis auf externen Player gelöscht.